_pages/decide/comparative-analysis.md

```diff
@@ -10,23 +10,23 @@ A detailed review of existing experiences provided either by direct competitors
 
 ## Reasons to use it
 
-To identify competitors’ solutions that excel, are lacking, or are missing critical design elements. Comparative analysis can give new solutions a competitive edge by identifying areas of opportunity, gaps in experience offerings, and potential design patterns to adopt or avoid.
+To identify competitors’ solutions that excel, are lacking, or are missing critical design elements. Comparative analysis can give you a competitive edge by identifying opportunities, gaps in other services, and potential design patterns to adopt or avoid.
 
 ## Time required
 
-**Medium:** 1–2 hours per identified competitor. One hour to analyze each competitor. 4–8 hours total to generate a report.
+**Medium:** 1–2 hours to analyze and write a report about each competitor.
 
 ## How to do it
 
-1. Identify a list of services or agencies that would be either direct or related competitors to the service or client agency. Pare the list down to four or five.
+1. Identify a list of services that would be either direct or related competitors to your service. Pare the list down to four or five.
 
-2. Establish which heuristics you will use to evaluate each service or agency offering.
+2. Establish which heuristics you will use to evaluate each competing service.
```

Comment:
I agree that heuristics may not be accessible, but it is a bit specific, and I'm not sure method or technique captures it.  I think of it more like a design "rule of thumb." But in this context, it is like using a heuristic as evaluation criteria.

"Establish which criteria or heuristics you will use to evaluate each competing service"...?